<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="1" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-07-24T13:52:27 IST" name="Default suite" finished-at="2021-07-24T13:52:57 IST" duration-ms="30299">
    <groups>
    </groups>
    <test started-at="2021-07-24T13:52:27 IST" name="Default test" finished-at="2021-07-24T13:52:57 IST" duration-ms="30299">
      <class name="ttt.bootcamp.testcases.LoginPageTest">
        <test-method is-config="true" signature="getPrerequisites()[pri:0, instance:ttt.bootcamp.testcases.LoginPageTest@21507a04]" started-at="2021-07-24T13:52:26 IST" name="getPrerequisites" finished-at="2021-07-24T13:52:27 IST" duration-ms="123" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getPrerequisites -->
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:ttt.bootcamp.testcases.LoginPageTest@21507a04]" started-at="2021-07-24T13:52:27 IST" name="openBrowser" finished-at="2021-07-24T13:52:30 IST" duration-ms="3651" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method signature="invalidDataLogin()[pri:0, instance:ttt.bootcamp.testcases.LoginPageTest@21507a04]" started-at="2021-07-24T13:52:30 IST" name="invalidDataLogin" finished-at="2021-07-24T13:52:35 IST" duration-ms="4706" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidDataLogin -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:ttt.bootcamp.testcases.LoginPageTest@21507a04]" started-at="2021-07-24T13:52:35 IST" name="closeBrowser" finished-at="2021-07-24T13:52:35 IST" duration-ms="75" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:ttt.bootcamp.testcases.LoginPageTest@21507a04]" started-at="2021-07-24T13:52:35 IST" name="openBrowser" finished-at="2021-07-24T13:52:38 IST" duration-ms="2527" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method signature="validDataLogin(ttt.bootcamp.bo.CredentialsBO)[pri:0, instance:ttt.bootcamp.testcases.LoginPageTest@21507a04]" started-at="2021-07-24T13:52:38 IST" name="validDataLogin" data-provider="getLoginCredentials" finished-at="2021-07-24T13:52:43 IST" duration-ms="4882" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[CredentialsBO [username=charlie.king@hotmail.com, password=zh4sxnd0q5]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[User is not able to login. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: User is not able to login. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at ttt.bootcamp.testcases.LoginPageTest.validDataLogin(LoginPageTest.java:15)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validDataLogin -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:ttt.bootcamp.testcases.LoginPageTest@21507a04]" started-at="2021-07-24T13:52:43 IST" name="closeBrowser" finished-at="2021-07-24T13:52:43 IST" duration-ms="75" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:ttt.bootcamp.testcases.LoginPageTest@21507a04]" started-at="2021-07-24T13:52:43 IST" name="openBrowser" finished-at="2021-07-24T13:52:45 IST" duration-ms="2503" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method signature="validDataLogin(ttt.bootcamp.bo.CredentialsBO)[pri:0, instance:ttt.bootcamp.testcases.LoginPageTest@21507a04]" started-at="2021-07-24T13:52:45 IST" name="validDataLogin" data-provider="getLoginCredentials" finished-at="2021-07-24T13:52:50 IST" duration-ms="4464" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[CredentialsBO [username=fred.kautzer@hotmail.com, password=zyahoscr4]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[User is not able to login. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: User is not able to login. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at ttt.bootcamp.testcases.LoginPageTest.validDataLogin(LoginPageTest.java:15)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validDataLogin -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:ttt.bootcamp.testcases.LoginPageTest@21507a04]" started-at="2021-07-24T13:52:50 IST" name="closeBrowser" finished-at="2021-07-24T13:52:50 IST" duration-ms="74" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:ttt.bootcamp.testcases.LoginPageTest@21507a04]" started-at="2021-07-24T13:52:50 IST" name="openBrowser" finished-at="2021-07-24T13:52:52 IST" duration-ms="2434" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method signature="validDataLogin(ttt.bootcamp.bo.CredentialsBO)[pri:0, instance:ttt.bootcamp.testcases.LoginPageTest@21507a04]" started-at="2021-07-24T13:52:52 IST" name="validDataLogin" data-provider="getLoginCredentials" finished-at="2021-07-24T13:52:57 IST" duration-ms="4475" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[CredentialsBO [username=clint.schroeder@gmail.com, password=4nsj8acu35m18hi]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[User is not able to login. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: User is not able to login. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at ttt.bootcamp.testcases.LoginPageTest.validDataLogin(LoginPageTest.java:15)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validDataLogin -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:ttt.bootcamp.testcases.LoginPageTest@21507a04]" started-at="2021-07-24T13:52:57 IST" name="closeBrowser" finished-at="2021-07-24T13:52:57 IST" duration-ms="73" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- ttt.bootcamp.testcases.LoginPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
